src/utils/utils.ts

```diff
@@ -1,22 +1,31 @@
 import type { WmLanguageCode } from '../types/options.js'
 
-// Ex: keep only 'fr' in 'fr_FR'
+/** Example: keep only 'fr' in 'fr_FR' */
 export function shortLang (language: string): WmLanguageCode {
   const lang = language.toLowerCase().split('_')[0] as WmLanguageCode
   return lang
 }
 
-// a polymorphism helper:
-// accept either a string or an array and return an array
-export function forceArray (array): string[] {
-  if (typeof array === 'string') array = [ array ]
-  return array || []
+/**
+ * a polymorphism helper:
+ * accept either a string or an array and return an array
+ */
+export function forceArray<T extends string> (array: T | readonly T[] | undefined): T[] {
+  if (typeof array === 'string') {
+    return [ array ]
+  }
+
+  if (Array.isArray(array)) {
+    // TODO: return readonly array
+    return [ ...array ]
+  }
+
+  return []
 }
 
-// simplistic implementation to filter-out arrays
-export const isPlainObject = (obj: any): boolean => {
-  if (!obj || typeof obj !== 'object' || obj instanceof Array) return false
-  return true
+/** simplistic implementation to filter-out arrays */
```

Comment:
[we are really on critical stuff here ;)]
Do you have some formatter on for comments? What's you're rule for using `//` or `/**`?

Reply:
/** comments are used as JSDoc and can be seen when hovering over methods. When writing code they are sometimes helpful to go over your code and hover over a library method to see more details about it.

// comments are just comments when looking into the file itself.

When I want others to read it when understanding methods, I use JSDoc comments. When they don’t matter to the user and are things like todo or important for the future dev like "it needs to be this way because X" I use // comments.

Reply:
I do like the "question everything" aspect. Getting challenged to overthink some ways of doing things is great. Sometimes there are other ways of doing things, sometimes someone is just used to some things so it’s a great way to improve :)

Reply:
Questioning things I'm not familiar with is part of my learning process, and I'm learning a lot with those PRs: thanks for sharing! :D